.babelrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This change is welcoming. Most of the text editors these days automatically pick up babelrc files and provide autocompletion features based on babel configuration.